packages/model-viewer/src/features/scene-graph/material.ts

```diff
@@ -66,57 +66,69 @@ export class Material extends ThreeDOMElement implements MaterialInterface {
 
     let {normalTexture, occlusionTexture, emissiveTexture} = gltfMaterial;
 
-    const normalTextures = new Set<ThreeTexture>();
-    const occlusionTextures = new Set<ThreeTexture>();
-    const emissiveTextures = new Set<ThreeTexture>();
+    let normalTextures: ThreeTexture|null = null;
+    let occlusionTextures: ThreeTexture|null = null;
+    let emissiveTextures: ThreeTexture|null = null;
```

Comment:
Since these are no longer sets, perhaps they should be singular?

Reply:
Done.